travis.sh

```diff
@@ -43,14 +43,28 @@ function error {
 
 trap error ERR
 
-git branch --all
-if [ "`git diff origin/master FETCH_HEAD .travis`" != "" ] ; then
-  DIFF=`git diff origin/master FETCH_HEAD .travis | grep .*Subproject | sed s'@.*Subproject commit @@' | sed 'N;s/\n/.../'`;
-  (cd .travis/;git log --oneline --graph --left-right --first-parent --decorate $DIFF) | tee /tmp/$$-travis-diff.log;
-  grep -c '<' /tmp/$$-travis-diff.log && exit 1;
-  echo "ok";
+
+travis_time_start is_jsk_travis_upgraded
+
+# Check if jsk_travis is upgraded, because downgrading jsk_travis is not supported.
+if [ "`git diff origin/master HEAD .travis`" != "" ] ; then
```

Comment:
@k-okada I think the diff should be between origin/master and HEAD. Why did you set FETCH_HEAD?